src/pages/InterviewSchedulingPage/components/InterviewScheduler.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import ScheduleSelector from 'react-schedule-selector';
+
+interface InterviewSchedulerProps {
+  numDays?: number;
+  minTime?: number;
+  maxTime?: number;
+  startDate?: Date;
+  reset?: boolean;
+  setDirtyCallback?: () => void;
+  selectedScheduleCallback?: (arg: Date[]) => void;
+}
+
+function InterviewScheduler({
+  numDays = 5,
+  minTime = 11,
+  maxTime = 21,
+  startDate = new Date(),
+  reset = false,
+  setDirtyCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+  selectedScheduleCallback = () => {}, // eslint-disable-line @typescript-eslint/no-empty-function
+}: InterviewSchedulerProps): JSX.Element {
+  const [selectedSchedule, setSelectedSchedule] = useState<Date[]>([]);
```

Comment:
we don't have to keep track of state within each interview scheduler if you're going to keep track of the scheduler state from the parent anyways
it's more idiomatic to keep track of state in parent, pass down schedule and callback as prop to this component etc.

Reply:
Not sure if this can be done. I encountered a pretty bad race condition when I tried doing this. I think each scheduler will have to have their own states.

Reply:
@godwinpang